addons/l10n_gcc_invoice/models/account_move.py

```diff
@@ -49,6 +49,9 @@ def _compute_narration(self):
         # Only update translations of real records
         self.filtered('id')._load_narration_translation()
 
+    def should_print_narration_english_translation(self):
```

Comment:
this should probably reference either l10n_gcc_invoice or arabic

eg. `def _l10n_gcc_invoice_print_narration_english_translation` or `def english_narration_different_than_arabic`

also if it was accepted, it might make sense to have a generic method instead, something like:

```py
def _l10n_gcc_invoice_english_different_than_arabic(self, field_name):
            return self.with_context(lang='ar_001')[field_name] != self.with_context(lang='en_US')[field_name]
```

Reply:
@nle-odoo As the narration is entered by the user, it would often not be translated. We could force the field to be translated by the user with the same function.
When messaging the PO he answered "if there is a translation, we can show the 2, else only the Arabic one". So I guess this is the thing to do.